angrybirds/stream.py

```diff
@@ -1,13 +1,15 @@
 import tweepy
 import json
+from .mongo import insert_document
 
 
 class AngryBirdsStreamListener(tweepy.StreamListener):
 
     def on_status(self, status):
         if "retweeted_status" in status._json:
             return
-        print(json.dumps(status._json, sort_keys=True, indent=4))
+        insert_document(status._json, 'tweets')
+        print(f'{status._json["user"]["screen_name"]}: {status._json["text"]}')
```

Comment:
Formatted strings like this currently break flake8, however a release to add Python 3.6 support should be made shortly https://gitlab.com/pycqa/flake8/issues/300